dynamic_rest/filters.py

```diff
@@ -283,7 +285,9 @@ def _filters_to_query(self, includes, excludes, serializer, q=None):
         def rewrite_filters(filters, serializer):
             out = {}
             for k, node in six.iteritems(filters):
-                filter_key = node.generate_query_key(serializer)
+                filter_key, field = node.generate_query_key(serializer)
+                if isinstance(field, (BooleanField, NullBooleanField)):
+                    node.value = node.value.lower() not in self.FALSEY_STRINGS
```

Comment:
Sure. Would you be ok if I did that in a follow-up PR? There's a P1 bug that's waiting for this fix so I'd like to get this in first.